test/test_unit_strings.cpp

```diff
@@ -84,6 +84,11 @@ TEST(unitStrings, si)
     EXPECT_EQ(to_string(ft * lb), "ft*lb");
 }
 
+TEST(unitStrings, laboratory_arb_units)
```

Comment:
```suggestion
TEST(unitStrings, laboratoryArbUnits)
```

This should fix the clang tidy issue